chapters/equations.tex

```diff
@@ -817,13 +817,20 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
 \lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
-automatically such that the resulting system is structurally nonsingular.  In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user
-expects.  A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without
-informing the user.  For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly
-used on these variables, do not have an effect on the simulation.
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.
 \end{nonnormative}
 
+Based on these non-normative considerations we have the following rules:
+A tool may add or remove initial equations automatically according to the rules below such that the resulting system is structurally nonsingular:
+\begin{itemize}
+\item A missing initial value of a discrete-time variable (see \cref{component-variability-prefixes-discrete-parameter-constant} -- this does not include parameter and constant variables) which does not influence the simulation result, may be automatically set to the start value or its default without informing the user.
```

Comment:
This way of writing looks fine to me:
- The dash is contained within a balanced pair of parentheses.
- It offers a way of breaking up the contents of the parenthesis in a relatively strong fashion without the need to turn to two full-fledged sentences, which would be problematic for a parenthesis in the middle of a sentence.
- We follow the "spaced en-dash" style rather than "spaceless em-dash", and we have somewhere around 80 of them in the document.

Reply:
@henrikt-ma, OK, let's use en dash then. What we should not do is to start the clause with an en dash and end with a parenthesis, which is what is written currently.

Reply:
Again, this is not how I read it; this is what I see _within the parentheses_:
> see \cref{component-variability-prefixes-discrete-parameter-constant} -- this does not include parameter and constant variables

Reply:
Ah, I see. Is there a way we can break up this sentence into 2 or rewrite it in some other way? This combination of brackets and en-dashes requires multiple re-reads to understand what it says, and even then prone to misinterpretation. Before posting my original comment I used my standard test for "Is it just me or is it really confusing?" which consists of grabbing a person next to me and asking "Does it make sense to you?" and together we tried to match the brackets and dashes and failed. So it is not just me.
For example:
> If a discrete-time variable (see \cref{component-variability-prefixes-discrete-parameter-constant}) that is not a constant or a parameter is missing an initial value and does not influence the simulation result, the initial value may be automatically set to the start value or its default without informing the user.

Hmm, trying to rewrite this, I have more questions about what we actually try to say here. 
1. Is it the variable that does not influence the simulation result or its initial value? How would you determine that?
2. In "set to the start value or its default": Is it default of the start value or the variable? Also, if the variable is missing an initial value, doesn't it mean that there is no start value?